manifests/r/Rime/Weasel/0.14.3/Rime.Weasel.installer.yaml

```
@@ -6,5 +6,6 @@ Installers:
   InstallerType: nullsoft
   InstallerUrl: https://github.com/rime/weasel/releases/download/0.14.3/weasel-0.14.3.0-installer.exe
   InstallerSha256: DE63DC2EF45D5B094A239FC704051270F3185BF9E883132532B7E17D2FDAF81B
+  ProductCode: Weasel
```

```suggestion
  ProductCode: Weasel
  AppsAndFeaturesEntries:
  - DisplayVersion:
    Publisher:
```
For matching with Registry values

---

That is why I added them as `null`. If they are not added, WinGet will map PackageVersion to DisplayVersion and think that this package writes `0.14.3` in Registry which is incorrect. These fields tell WinGet that the application does not set these values to Registry.